manual/manual/cmds/instrumented-runtime.etex

```diff
@@ -0,0 +1,332 @@
+\chapter{Runtime tracing with the instrumented runtime}
+%HEVEA\cutname{instrumented-runtime.html}
+
+This chapter describes the OCaml instrumented runtime, a runtime variant
+allowing the collection of events and metrics.
+
+Collected metrics include time spent executing the {\em garbage collector},
+allowing the overall execution time of individual pauses to be measured
+down to the time spent in specific parts of the garbage collection algorithm.
+
+Insight is also given on memory allocation and motion, recording
+the size of allocated memory blocks, as well as value promotions from the
+{\em minor heap} to the {\em major heap}.
```

Comment:
Maybe the first comma could be replaced by `by` to avoid mixing two kind of commas

> Insight is also given on memory allocation and motion by recording
the size of allocated memory blocks, as well as value promotions from the
{\em minor heap} to the {\em major heap}.

?